doc/plutus-core-spec/introduction.tex

```diff
@@ -18,3 +18,51 @@ \section{Introduction}
 untyped Plutus Core is used as a compilation target, and we will eventually
 provide a specification of the type system and semantics of Typed Plutus Core
 here as well, together with its relationship to Untyped Plutus Core.
+
+\subsection{Terminology and version numbers}
+\label{sec:version-numbers}
+Once a Plutus Core script has been deployed to the blockchain, it may be
+re-evaluated at any point in the future when the history of the chain is
+verified.  Plutus Core may evolve as time progresses, and because of this we
+have to be quite careful with versioning.  A detailed discussion of this issue
+can be found in~\cite{CIP-35}, but we reproduce two definitions here for
+convenience.
+\begin{itemize}
+\item The language itself may evolve: for example some fundamental new construct
+  may be added.  A \textit{Plutus Core language version} describes a version of
+  the basic language with a particular set of features. A language version
+  consists of three non-negative integers separated by decimal points, for
+  example \texttt{1.4.2}.  Language versions are ordered lexicographically.
+\item The built-in types and functions deployed on the chain along with a
+  particular language version can change with time, and we refer to Plutus Core
+  along with a particular set of types and functions as a \textit{Plutus Core
+    ledger language version}.  These are referred to using version names like
+  ``Plutus V1,'', ``Plutus V2'', and so on.
+  \end{itemize}
+ \noindent These two kinds of version are independent: all Plutus Core language
+ versions must support all ledger language versions.
+
+\subsection{Cardano support for different Plutus Core language and ledger language versions}
+
+Cardano introduced support for Plutus Core in the Alonzo release, and
+Table~\ref{table:versions} shows later releases which have introduced new
```

Comment:
Suggestion: "shows later releases" => "shows all releases that have introduced new language versions and/or new ledger language versions"